geomstats/geometry/symmetric_matrices.py

```diff
@@ -19,9 +19,13 @@ class SymmetricMatrices(EmbeddedManifold):
     """
 
     def __init__(self, n, **kwargs):
+        matrices = Matrices(n, n)
         super(SymmetricMatrices, self).__init__(
             dim=int(n * (n + 1) / 2),
-            embedding_manifold=Matrices(n, n))
+            embedding_manifold=matrices,
+            ambient_manifold=matrices,
+            submersion=matrices.to_skew_symmetric,
+            tangent_submersion=matrices.to_skew_symmetric)
         self.n = n
 
     def belongs(self, mat, atol=gs.atol):
```

Comment:
Shouldn't `belong` be called through `EmbeddedManifold` here?

Reply:
Yes :)